web-components/src/components/taskitem/TaskItem.ts

```diff
@@ -30,6 +29,43 @@ export namespace TaskItem {
     @property({ type: String }) customAriaLabel = "";
     @property({ type: String }) iconSrc = "";
     @property({ type: String }) tabIndexForContainer = "0";
+    @property({ type: String }) titleValue = "";
+    @property({ type: String }) itemTitleValue = "";
+
+    /**
+    * @deprecated Use `itemTitle` instead.
+    * Using title may cause an unwanted tooltip.
+    */
+    @property({ type: String, reflect: true})
+    get title(): string {
+      return this.displayTitle;
+    }
+    set title(title: string) {
+      this.titleValue = title;
+      this.updateTitle();
+    }
+    /**
+    * The visible text for the Title.
+    */
+    @property({ type: String, reflect: true, attribute: "itemtitle" })
+    get itemTitle() {
+      return this.displayTitle;
+    }
+    set itemTitle(itemTitle: string) {
+      this.itemTitleValue = itemTitle;
+      this.updateTitle();
+    }
+
+    private updateTitle() {
+      if(!this.itemTitleValue){
+      this.displayTitle = this.titleValue;
+      }
+      else{
+        this.displayTitle = this.itemTitleValue;
```

Comment:
We just prioritise itemTitle for display text. But will use title if nothing is provided.

Reply:
removing title is a breaking API change. this should be marked as deprecated and left

Reply:
This doesn't remove it though, sorry if I'm not following, any legacy usage is preserved in this PR.